modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/components/_field-options.scss

```diff
@@ -41,6 +41,7 @@ $c: '.ddm-field-options';
 				background: transparent;
 				border: none;
 				color: #6b6c7e;
+				cursor: default;
```

Comment:
@alinedoleron, shouldn't the `cursor` be `not-allowed` instead of `default`? Have you discussed with Felipe Pires about that? Actually, depending of the value of `readOnlyFieldName`, the cursor should assume different values (`auto/default` or `not-allowed`).